V5 release note - Automation part (#871)
akeneo/pim-helpcenter: content/updates/automation/5-Smoother-rule-execution-monitoring.md

# Smoother rule execution monitoring
::: meta-data type="Improvement" ee-only="true"
:::

> From now on, every time you run a rule, a rule execution operation row will appear in your dashboard under the latest operations as well as in the process tracker. As with any other job or operation, you can access the detailed execution page with one click. This execution page will inform you if the rule went well or if it generated any errors or warnings, and will tell you how to correct them.

Reviewer: ```suggestion
Every time you run a rule, a rule execution operation row appears in your dashboard under the latest operations as well as in the process tracker. As with any other job or operation, the detailed execution page is only one click away. This execution page will inform you if the rule went well or if it generated any errors or warnings, and will tell you how to correct them.
```

> From now on, every time you run a rule, a rule execution operation line will appear in your dashboard under the last operations as well as in the process tracker. Like for any other job or operation, you can access the detailed execution page with a click. This execution page will inform you if the rule went fine or if it raised any errors or warnings, and give you guidance on how to fix them.

Reviewer 2: ```suggestion
From now on, every time you run a rule, a rule execution operation row will appear in your dashboard under the latest operations as well as in the process tracker. As with any other job or operation, you can access the detailed execution page with one click. This execution page will inform you if the rule went well or if it generated any errors or warnings, and will tell you how to correct them.
```
